src/Formality.rei

```diff
@@ -18,13 +17,15 @@ type validateAsync('value, 'message) =
 
 type validator('field, 'value, 'state, 'message) =
   Formality__Validation.validator('field, 'value, 'state, 'message) = {
+    field: 'field,
     strategy: Formality__Strategy.t,
     dependents: option(list('field)),
```

Comment:
If `dependents` is `option(list('a))` isn't it easier to keep just `list('a)`? If no dependent fields is required it would be just empty list? I don't know what are performance drawbacks but it would simplify the API.

Reply:
I chose `option(list())` b/c it seems more idiomatic and less implicit but maybe I'm wrong here. Asked opinions on Discord. If you don't mind, I'll merge it as is and if people think that unwrapped list is better then I will change it in the next PR, which solves `value` problem.

Reply:
I commented in #35 about how I'm using the library so it's not a big issue for me.